mmtrack/models/track_heads/stark_head.py

```diff
@@ -0,0 +1,376 @@
+import torch
+import torch.nn.functional as F
+from mmcv.cnn.bricks import ConvModule
+from mmdet.core.bbox.transforms import bbox_xyxy_to_cxcywh
+from mmdet.models import HEADS
+from mmdet.models.dense_heads import DETRHead
+from mmdet.models.utils import Transformer
+from mmdet.models.utils.builder import TRANSFORMER
+from torch import nn
+
+@HEADS.register_module()
+class CornerPredictorHead(nn.Module):
+    """Corner Predictor module.
+
+    Args:
+        inplanes (int): input channle
+        channel (int): the output channel of the first conv block
+        feat_size (int): the size of feature map
+        stride (int): the stride of feature map from the backbone
+    """
+
+    def __init__(self,
+                 inplanes=64,
+                 channel=256,
+                 feat_size=20,
+                 stride=16):
+        super(CornerPredictorHead, self).__init__()
+        self.feat_size = feat_size
+        self.stride = stride
+        self.img_size = self.feat_size * self.stride
+
+        def conv_module(in_planes,
+            out_planes,
+            kernel_size=3,
+            padding=1):
+            # The module's pipeline: Conv -> BN -> ReLU.
+            return ConvModule(
+                in_channels=in_planes,
+                out_channels=out_planes,
+                kernel_size=kernel_size,
+                padding=padding,
+                bias=True,
+                norm_cfg=dict(type='BN', requires_grad=True),
+                act_cfg=dict(type='ReLU'),
+                inplace=True)
+
+        # top-left corner
+        self.conv1_tl = conv_module(inplanes, channel)
+        self.conv2_tl = conv_module(channel, channel // 2)
+        self.conv3_tl = conv_module(channel // 2, channel // 4)
+        self.conv4_tl = conv_module(channel // 4, channel // 8)
+        self.conv5_tl = nn.Conv2d(channel // 8, 1, kernel_size=1)
```

Comment:
use ConvModule

Reply:
use ConvModule is like this: 
ConvModule(
                in_channels=in_planes,
                out_channels=out_planes,
                kernel_size=kernel_size,
                padding=padding,
                bias=True,
                norm_cfg=dict(type='BN', requires_grad=True)
then use nn.Sequential or nn.ModuleList to warp five that ConvModule may seem ugly.